app/views/procedures/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @procedures.each do |p|%>
+    <div><%=p.name%>
+    
+    <%p.facilities.each do |f|%>
```

Comment:
don't use single character variable names. Your variables should be descriptive.